### DISCUSSION TITLE
Community Edition in Citrix VDA

### DISCUSSION THREAD

**Discussion by sbrumley** [dbeaver/dbeaver#16014] [Q&A]
Hi a SQL developer in our company has requested I add DBeaver to our Citrix desktops.  I've installed the community edition (for all users) but when it tries to launch it fails, log file attached.  Is there anything special I need to do for a Citrix/RDS environment?  The session hosts are Windows Server 2016.
[noname.log.txt](https://github.com/dbeaver/dbeaver/files/8380403/noname.log.txt)

Thanks,
sbrumley

**Comment by LonwoLonwo** [comment]
Hello @sbrumley 

Same issue here https://github.com/dbeaver/dbeaver/issues/15117
